packages/beacon-node/src/network/reqresp/reqResp.ts

```diff
@@ -1,3 +1,4 @@
+import {setMaxListeners} from "node:events";
```

Comment:
Shame we have to introduce a nodejs specific api here.

Later we can pass down a signal from the top-level beacon node and avoid creating a new signal here.